plugins/tradeMetricsHandler.go

```diff
@@ -0,0 +1,42 @@
+package plugins
+
+import (
+	"github.com/stellar/kelp/model"
+)
+
+// TradeMetricsHandler tracks the number of trades
+type TradeMetricsHandler struct {
+	trades []model.Trade
+}
+
+// MakeTradeMetricsHandler is a factory method for the TradeMetricsHandler
+func MakeTradeMetricsHandler() *TradeMetricsHandler {
+	return &TradeMetricsHandler{
+		trades: []model.Trade{},
+	}
+}
+
+// Reset sets the handler's trade counter to zero.
+func (h *TradeMetricsHandler) Reset() {
+	h.trades = []model.Trade{}
+}
+
+// Read stores new trades internally.
+func (h *TradeMetricsHandler) Read(newTrades []model.Trade) {
```

Comment:
Instead of calling this function `Read` we should call it `HandleFill` so it can implement `api.FillHandler`
also it should take in a single `model.Trade` and not an array